### ISSUE TITLE
"time" object has no attribute sleep

### ISSUE THREAD

**Issue by lion24** [mtshiba/pylyzer#95]
Hi there 👋 

I probably getting something wrong, but I cannot spot the issue.

I did install using:
```
pip install pylyzer
```

Version:
```
PS > pylyzer --version
pylyzer 0.0.54
```

And I tried to against a simple file import a module from the standard library:
```
import time


def test_sleep():
    time.sleep(1)
    print("Slept for 1 second")


if __name__ == "__main__":
    test_sleep()
```

Running pylyzer lead to a "object has no attribute sleep" error:
```
pylyzer .\sleep.py
Start checking: sleep.py
Found 1 errors: sleep.py
Error[#1393]: File .\sleep.py, line 5, <module>.test_sleep

5 |     time.sleep(1)
  :          -----

AttributeError: PyModule("c:\\users\\lion24\\.erg\\lib\\pystd\\time.d.er") object has no attribute sleep
```

In the meantime `time.d.er` looks good:
```
PS > type c:\users\lion24\.erg\lib\pystd\time.d.er\time.d.er
.sleep!: Float => NoneType
.time!: () => Float

.StructTime = 'struct_time': ClassType
.StructTime.
    tm_year: Nat
    tm_mon: Nat
    tm_mday: Nat
    tm_hour: Nat
    tm_min: Nat
    tm_sec: Nat
    tm_wday: Nat
    tm_yday: Nat
    tm_isdst: Nat
    tm_zone: Str
    tm_gmtoff: Nat
```

And I'm running on Windows 10.

**Comment by lion24** [comment]
I checked on Linux and it works fine. So I guess this is an error related to windows.

**Comment by mtshiba** [comment]
The part that seemed to be the problem has been fixed in https://github.com/mtshiba/pylyzer/commit/974269a2ba48d4afcd4a65e3fd55209680ae585d.

Please uninstall pylyzer and reinstall it.

**Comment by lion24** [comment]
> The part that seemed to be the problem has been fixed in [974269a](https://github.com/mtshiba/pylyzer/commit/974269a2ba48d4afcd4a65e3fd55209680ae585d).
> 
> Please uninstall pylyzer and reinstall it.

Yup it works ! 

A small quirk, I manually had to remove the `.erg` folder from the user directory, I guess because when doing uninstallation, files are left intact.

But now it works fine :)
